content/applications/websites/ecommerce/managing_products/products.rst

```diff
@@ -0,0 +1,197 @@
+apply to *all* product pages.
+
+Features
+--------
+
+.. _ecommerce-features:
+
+In the **website builder** window, click :guilabel:`Customize` to enable additional features such as
+:guilabel:`Rating`, :guilabel:`Share`, etc.
+
+- :guilabel:`Customers`: :guilabel:`Rating` adds a rating option on the page; :guilabel:`Share` adds
+  share buttons via social media networks and email;
+- :guilabel:`Select Quantity`: if enabled, allows to choose the quantity added to cart;
+- :guilabel:`Tax Indication`: notifies if the price is **VAT included** or **excluded**;
+- :guilabel:`Variants`: shows all possible variants of the product as a :guilabel:`Products List`;
+  :guilabel:`Options` as selectable options to compose the variant yourself;
+- :guilabel:`Cart`: :guilabel:`Buy Now` adds a checkout button taking the customer directly to the
+  checkout page; :guilabel:`Wishlist` allows to add the product to a wishlist;
+- :guilabel:`Specification`: allows you to select where the **specification** category is displayed.
+   This only applies to products with **variants**.
+
+.. note::
+   - To allow **wishlists**, the option must be enabled in :menuselection:`Configuration -->
+     Settings --> Shop - Products --> Wishlists`;
+   - To access the :guilabel:`Variants` options, the :guilabel:`Product Variants` option must first
+     be enabled under :menuselection:`Configuration --> Settings --> Shop - Products`. You can learn
+     more about variants :doc:`here <../../../sales/sales/products_prices/products/variants>`.
+
+Layout
+------
+
+Within the same :guilabel:`Customize` tab as the :ref:`features <ecommerce-features>`, the layout
+configuration can be changed according to your needs.
+
+- :guilabel:`Images Width`: changes the *width* of the image product displayed on the page;
+- :guilabel:`Layout`: the :guilabel:`Carousel` layout displays a large, main image with smaller ones
+  underneath; whereas the :guilabel:`Grid` displays four images in a square layout (see pictures
+  below);
+- :guilabel:`Image Zoom`: if the zoom occurs on :guilabel:`Pop-up on Click`, when hovering over the
+  image (:guilabel:`Magnifier on hover`), on :guilabel:`Both`, or :guilabel:`None`;
+- :guilabel:`Image Spacing`: how far apart the images are;
+- :guilabel:`Columns`: how many columns of images are displayed;
+- :guilabel:`Main Image`: click :guilabel:`Replace` to change the product's main image;
+- :guilabel:`Extra Images`: click :guilabel:`Add` or :guilabel:`Remove all` to add or remove product
+  images. You can also add images and videos via URL.
+
+.. note::
+   Images must be either PNG or JPG and at least 1042px in height or width. To trigger the zoom, the
+   image has to be bigger than 1024x1024
+
+.. image:: products/products-layout.png
+   :align: center
+   :alt: Product images layout
+
+Add content
+-----------
+
+You can use **building blocks** to add content to your product page. These blocks can be used to add
```

Comment:
Where to select building blocks? Is there a "building blocks" option available somewhere? should we enable this feature? Could you please clarify? :)